### PR TITLE
fix(search): handle large numbers in search queries

### PR DESCRIPTION
**Pull request by closingin**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [x] Create automatic tests
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by arnaudbesnier** [comment]
Task linked: [CU-dyxfm6 P1 - Search - Searching on BIGINT field type returns no results (PostgreSQL)](https://app.clickup.com/t/dyxfm6)

File: .eslintrc.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     'sonarjs',
   ],
   env: {
+    es2020: true,
```
Comment (VincentMolinie) [inline .eslintrc.js:12]: Why ?

**Review by VincentMolinie — CHANGES_REQUESTED** [review]
Capture d’écran 2021-02-11 à 11 49 50
Does not work on my side. I think if you have another column that is not bigint it just crash 🤷

**Review by VincentMolinie — DISMISSED** [review]

**Comment by VincentMolinie** [comment]
A little test would be appreciated, anyway codeclimate won't let you merge if you don't 😉

**Comment by closingin** [comment]
It is currently impossible to test this fix correctly, as the [library](https://github.com/domasx2/sequelize-fixtures) we use to load fixtures does not support large numbers. Its reader [here](https://github.com/domasx2/sequelize-fixtures/blob/master/lib/reader.js#L16) relies on `JSON.parse`, which is not compatible with numbers over `MAX_SAFE_INTEGER`.

Using [`json-bigint`](https://www.npmjs.com/package/json-bigint) would fix it, but the project seems unmaintained. Maybe we should fork?

If not, then we'll have to merge without testing. I still pushed the required tests even though they do not work.

**Review by VincentMolinie — APPROVED** [review]

**Review by VincentMolinie — APPROVED** [review]

**Comment by closingin** [comment]
Someone needs to approve this PR on CodeClimate. Based on the current codebase, I'm kinda forced to keep it as it is right now.

**Pull request merged**